Api/Controllers/RestaurantGroupsControllers.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Reservant.Api.Identity;
+using Reservant.Api.Models;
+using Reservant.Api.Models.Dtos;
+using Reservant.Api.Models.Vmodels;
+using Reservant.Api.Services;
+using Reservant.Api.Validation;
+
+namespace Reservant.Api.Controllers
+{
+    /// <summary>
+    /// Controler for getting restaurantGroups of restaurant owner
+    /// Only RestaurantOwner can use this controller
+    /// </summary>
+    /// <request code="400"> Validation errors </request>
+    /// <request code="401"> Unauthorized </request>
```

Comment:
\<request> to tylko dla końcówek

(służy do opisania co oznaczają poszczególne kody)